Tweetinvi/ExceptionHandler.cs

```diff
@@ -15,121 +15,108 @@ namespace Tweetinvi
     /// </summary>
     public static class ExceptionHandler
     {
-        [ThreadStatic]
-        private static IExceptionHandler _exceptionHandler;
+        public static IExceptionHandler ExceptionHandlerInstance { get; }
 
-        /// <summary>
-        /// Current Thread ExceptionHandler
-        /// </summary>
-        public static IExceptionHandler CurrentThreadExceptionHandler
+        static ExceptionHandler()
         {
-            get
-            {
-                if (_exceptionHandler == null)
-                {
-                    Initialise();
-                }
-
-                return _exceptionHandler;
-            }
+            ExceptionHandlerInstance = TweetinviContainer.Resolve<IExceptionHandler>();
         }
 
         /// <summary>
         /// Notify that a WebException has been received and that the ExceptionHandler is going to process it.
         /// </summary>
-        public static event EventHandler<GenericEventArgs<ITwitterException>> WebExceptionReceived
-        {
-            add { CurrentThreadExceptionHandler.WebExceptionReceived += value; }
-            remove { CurrentThreadExceptionHandler.WebExceptionReceived -= value; }
-        }
-
-        static ExceptionHandler()
-        {
-            Initialise();
-        }
-
-        private static void Initialise()
+        public static event EventHandler<GenericEventArgs<ITwitterException>> TwitterExceptionRaised
         {
-            _exceptionHandler = TweetinviContainer.Resolve<IExceptionHandler>();
+            add { ExceptionHandlerInstance.TwitterExceptionRaised += value; }
+            remove { ExceptionHandlerInstance.TwitterExceptionRaised -= value; }
         }
 
         /// <summary>
         /// DEFAULT VALUE : TRUE. If set to true, WebException will no longer throw. Instead you will receive a result of null/false/0.
         /// </summary>
-        public static bool SwallowWebExceptions
+        public static bool OnTwitterExceptionReturnNull
         {
-            get { return CurrentThreadExceptionHandler.SwallowWebExceptions; }
-            set { CurrentThreadExceptionHandler.SwallowWebExceptions = value; }
+            get { return TweetinviConfig.ApplicationSettings.OnTwitterExceptionReturnNull; }
+            set { TweetinviConfig.ApplicationSettings.OnTwitterExceptionReturnNull = value; }
         }
 
         /// <summary>
         /// DEFAULT VALUE : TRUE. If set to true, Exceptions will be Logged and you will be able to access them from the ExceptionHandler.
```

Comment:
As commented elsewhere, can we change the default `LogExceptions` value to false?